src/assets/scripts/client/trafficGenerator/buildPreSpawnAircraft.js

```diff
@@ -92,25 +92,31 @@ const _calculateDistancesAlongRoute = (waypointModelList, airport) => {
     let distanceFromClosestFixToAirspaceBoundary = 0;
     let totalDistance = 0;
 
-    for (let i = 0; i < waypointModelList.length; i++) {
+    // Iteration started at index 1 to ensure two elements are available. It is
+    // already an expectation that aircraft must have two waypoints, so this
+    // should not be a problem here.
+    for (let i = 1; i < waypointModelList.length; i++) {
         const waypoint = waypointModelList[i];
-        const waypointPosition = waypoint.relativePosition;
-        let previousWaypoint = waypoint;
-        let previousPosition = waypoint.relativePosition;
+        const previousWaypoint = waypointModelList[i - 1];
 
-        if (i > 0) {
-            previousWaypoint = waypointModelList[i - 1];
-            previousPosition = previousWaypoint.relativePosition;
+        // TODO: Refactor to avoid continue statement
+        if (waypoint.isVector || previousWaypoint.isVector) {
+            continue;
```

Comment:
@erikquinn I'm really not a fan of the `no-continue` rule.  In fact, I think I changed it to a warning in one of my branches 🤔 

I understand the sprit of the rule.  My thoughts are more like this, though:
```javascript
// rule suggests
for (let i = 0; i< length; i++) {
  if (condition) {
    // do stuff
  }
}

// I much prefer
for (let i = 0; i< length; i++) {
  if (!condition) {
      continue;
  }

  // do stuff
}

```